apps/aptos/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -115,6 +116,8 @@ const IfoCardTokens: React.FC<React.PropsWithChildren<IfoCardTokensProps>> = ({
 
   const publicPoolCharacteristics = publicIfoData[poolId]
   const userPoolCharacteristics = walletIfoData[poolId]
+  const { amountTokenCommittedInLP, refundingAmountInLP } = userPoolCharacteristics
+  const spentAmount = amountTokenCommittedInLP.minus(refundingAmountInLP)
```

Comment:
Let's move `amountTokenCommittedInLP.minus(refundingAmountInLP)` inside here to avoid unnecessary calcualtion when ifo is not live.

```
value={getBalanceNumber(amountTokenCommittedInLP.minus(refundingAmountInLP), currency.decimals)}
```

Reply:
Done